components/director/internal/formationmapping/handler.go

```diff
@@ -127,6 +131,33 @@ func (h *Handler) UpdateStatus(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		if isFADeleted {
```

Comment:
what is the idea of this `if`? It just extract part of the `h.processFormationAssignmentAsynchronousUnassign` into the handler function without any changes? Or I'm missing something? And the transaction seems to be the same